inst/examples/app/analytics.R

```diff
@@ -1,24 +1,12 @@
 library(shiny)
+library(dplyr) # necessary to import %>%
 library(RSQLite)
 # please install shiny.telemetry with all dependencies
 # install.packages("shiny.telemetry", dependencies = TRUE)
 library(shiny.telemetry)
 
-# prepare credentials list to access logs:
-db_credentials <- list(
-  DB_NAME = "user_stats.sqlite",
-  DB_DRIVER = "SQLite"
+data_storage <- DataStorageRSQLite$new(
+  username = "viewer", db_path = "user_stats.sqlite"
 )
 
-# define function hot to get username
-get_user <- function(session) {
-  username <- isolate(parseQueryString(session$clientData$url_search)$username)
-  req(username)
-  return(username)
-}
-
-# define ui and server
-ui <- shiny_stats_ui()
-server <- shiny_stats_server(get_user, db_credentials = db_credentials)
-
-shinyApp(ui = ui, server = server, options = list(port = 8887, launch.browser = TRUE))
+run_analytics_dashboard(data_storage)
```

Comment:
When running the instrumented app and the analytics in sequential order the analytics dashboard crashes

Am I doing something wrong?

I've tried re-installing the package and replacing `library(shiny.telemetry)` with `devtools::load_all()`

[Kooha-2023-03-29-00-37-27.webm](https://user-images.githubusercontent.com/211358/228382386-cfd230db-7ac1-4e20-8418-0893d898f178.webm)

Reply:
I was able to recreate the error. 

I think the issue is where the `.sqlite` file is. I had the `.sqlite` file in my root. When I moved it to `inst/examples/app`, the analytics dashboard crashed with the same error (I think). I then redefined the `data_storage` with a new path to the `.sqlite` file, i.e.

```
data_storage <- DataStorageRSQLite$new(
    username = "viewer", db_path = "inst/examples/app/user_stats.sqlite"
)
```

The analytics dashboard started working again. Please try if this works.

Reply:
Regarding the updates to `instrumented-app.R`, I already copied the `get_user()` function with default username and added a `log_brower_version()`. However, it seems like the `log_browser_version()` does not work with either sql lite or log file. My suggestion would be to solve the issue with `log_browser_version()` in a separate PR.